Makefile

```diff
@@ -168,6 +168,7 @@ kops: ${KOPS}
 ${KOPS}: ${BINDATA_TARGETS}
 	go build ${GCFLAGS} ${EXTRA_BUILDFLAGS} ${LDFLAGS}"-X k8s.io/kops.Version=${VERSION} -X k8s.io/kops.GitVersion=${GITSHA} ${EXTRA_LDFLAGS}" -o $@ k8s.io/kops/cmd/kops/
 
+.PHONY: ${GOBINDATA}
```

Comment:
Why? This task actually creates/updates the file that `${GOBINDATA}` expands to, so it's not phony.

Reply:
You're right - fixed!